README.md

```diff
@@ -1,2 +1,30 @@
 # hpccarpentry-mini-intro
-material for our course on hpccarpentry
+material for our course on hpccarpentry.
+
+There were max 16 people last week so we can probably expect the same. This would mean a max of 3 breakout rooms.
+
+# Agenda
+## What is HPC Carpentry? (20 mins, including discussion)
+* Introduction to HPC Carpentry and what it's origins and goals are
+* What HPC Carpentry is *not*
+## (Inspired ideas from Peter go here)
+
+## Lesson development for Centres of Excellence
+FocusCoE has funded the creation of a template lesson on how to run a community code **at scale**. The idea being that such a lesson might be useful to CoEs who typically have a marquee scalable application. We chose LAMMPS as the test case because it is used heavily by the E-CAM community and development on top of LAMMPS is very common. 
+
+### Overview of lesson design (20 minutes)
```

Comment:
This could be shorter but I wanted to leave time to debate a bit.

Reply:
Here I wanted to fill in some details, so that you get an overview what I intend to do. I am waiting for this until we merge the document.

Reply:
I was just going to talk specifically about my lesson here, but if you want to have a more cross-cutting discussion (with more meat), then we should pull this out from the subheading and give it more prominence.

Reply:
Tell you what, edit this however you see fit and then we sync up to thrash out who will do what. Note that the guys had one break an hour (see their agenda at https://coderefinery.github.io/2020-11-02-instructor-training/), so in total we only really have 3hr20 for instruction

Reply:
sounds good to me. Let's merge it then.